182 - Set up file structure for the new microservice

Kataruse opened the pull request:
This is a combined effort between me, JL, and Ryan with tickets 191-193 with us each branching of each others work as dependencies pass.

Essentially boils down to the GameAPI code being exactly copied to the SubmissionAPI file with any references directly to GameAPI being switched and the proper ports in all locations being changed. With testing this runs locally, in AWS, and by itself without any other microservice up. Docker now actually builds the Submission API and says it's up and running along with the associated volume.

Also merged in Blaine's 138 to prevent any merging issues if they occur.

Commit:
Added a folder for the submission microservice as well as a solution (same thing as the rest of the microservices). The name for the folder can be changed if needed. I also threw a ReadMe into the folder just to have another file in it. It can be deleted when there's other files put into it.

rhanc committed:
these are not the drones you are looking for

Kataruse committed:
I think everything is fully working?

Kataruse committed:
Accidentally changed enviornment for submission gateway with docker prod deployment

Kataruse committed:
Merge pull request #124 from Andersonjb1/138-Blaine

138 blaine

T0xen reviewed (commented):
Dude go to sleep 😭

Kataruse commented:
Nah buh I'm locked In

<img width="220" height="220" alt="image" src="https://github.com/user-attachments/assets/df7ac982-9e9a-4d2a-b627-caaf19b9639b" />

Andersonjb1 commented on Team-QWERTY-BucStop_Submission/SubmissionAPIGateway/appsettings.containers.json:
Might need to re-look these config files since they're still using the public urls.

Andersonjb1 commented on Team-QWERTY-BucStop_Submission/SubmissionAPIGateway/appsettings.containersLocal.json:
Same here with localhost.

Andersonjb1 commented on docker-compose.yml:
If its ready then copy how the other containers are built in the production compose.

Kataruse committed:
Update docker-compose to use submission data volume

Replaced local folder for game suggestions with submission data volume and commented out build instructions for submission-gateway.

JoeNeglia reviewed (approved):
GREEN?
Everything still works locally. I don't really know what else to look for ngl.

Andersonjb1 reviewed (approved):
## GREEN 
# 🙏🏻

Andersonjb1 committed:
Merge branch 'Dev-6' into Nick-193

Andersonjb1 reviewed (approved):
# GREEN
